### PR TITLE
Added TypeScript Install instructions

### PR DESCRIPTION
**Pull request by DenisCarriere**
Once the PR is merged to DefinitelyTyped

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/10921

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,14 @@ split it into 4 children cells and put them in the queue.
 It will be guaranteed to be a global optimum within the given precision.
 
 ![image](https://cloud.githubusercontent.com/assets/25395/16748630/e6b3336c-47cd-11e6-8059-0eeccf22cf6b.png)
+
+
+### TypeScript
+
+TypeScript implementation was added to the [DefinitelyTyped repository](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/88fca07022ae6549a09965bf1316a1f5e34e471b/js-base64/js-base64.d.ts).
```
Comment (TWiStErRob) [inline README.md]: Url points to base64 not polylabel

Comment (DenisCarriere) [inline README.md]: Ops... Just realized that, I'll close this PR and re-submit

**Comment by TWiStErRob** [comment]
Tip for future: this is the exception when you can and should force push an amended commit. Or commit new ones and squash all of the together. GitHub will pick up the change in the same PR. Of course you can't do that from the web.

**Comment by DenisCarriere** [comment]
Thanks for the tip.

**Pull request closed without merging**